src/view/edit-event-view.js

```diff
@@ -2,6 +2,10 @@ import dayjs from "dayjs";
 import {EventType, ADDITIONAL_OFFERS} from "../helpers/constants";
 import Smart from "./smart";
 import {getDestination} from "../mock/trip-event";
+import flatpickr from "flatpickr";
+
+import "../../node_modules/flatpickr/dist/flatpickr.min.css";
```

Comment:
Здесь относительный пусть должен работать 
```suggestion
import "flatpickr/dist/flatpickr.min.css";
```